### DISCUSSION TITLE
Where can i get yesterday live lectures on the date of 1/25/2023?

### DISCUSSION THREAD

**Discussion by talhash02** [AammarTufail/python_ka_chilla2023#91] [Q&A]
Where can i get yesterday live lectures on the date of 1/25/2023?

**Comment by Muhammad-Ali-Butt** [comment]
it didnt go live since it was a surprise practice session where @AammarTufail asked us to do everything we learned so far on a covid 19 dataset and made us send the pdf of it to him on telegram before ending the class.
